Zork not installable, jzip fails

→ brew install zork1
==> Installing zork1 from wjwwood/zork
==> Installing dependencies for wjwwood/zork/zork1: jzip
==> Installing wjwwood/zork/zork1 dependency: jzip
==> Downloading http://download.sourceforge.net/jzip/jzip21-10oct2000.zip
Already downloaded: /Users/ssonne/Library/Caches/Homebrew/jzip-2.1.zip
==> Patching
patching file osdepend.c
==> make -f unixio.mak
Last 15 lines from /Users/ssonne/Library/Logs/Homebrew/jzip/01.make:
          ((g)->have ? ((g)->have--, (g)->pos++, *((g)->next)++) : gzgetc(g))
                        ~~~^ ~~~~
fileio.c:123:35: error: member reference base type 'gzFile' (aka 'struct gzFile_s *') is not a structure or union
         story_offset += 65536L * jz_getc( gfp );
                                  ^~~~~~~~~~~~~~
./ztypes.h:84:20: note: expanded from macro 'jz_getc'
#define jz_getc    gzgetc
                   ^
/usr/include/zlib.h:1699:41: note: expanded from macro 'gzgetc'
          ((g)->have ? ((g)->have--, (g)->pos++, *((g)->next)++) : gzgetc(g))
                                     ~~~^ ~~~
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [fileio.o] Error 1
make: *** Waiting for unfinished jobs....

--- a/jzip.rb
+++ b/jzip.rb
@@ -7,7 +7,7 @@ class Jzip < Formula
   version '2.1'
 
   def patches
-    DATA
+    :DATA
   end
 
   def install

--- a/zork1.rb
+++ b/zork1.rb
@@ -8,7 +8,7 @@ class Zork1 < Formula
   version '1'
   sha256 '645027fb189ba8fd30daecca609e68e2bf2337b76e7b4f925ccf58778f3eac40'
 
-  depends_on 'jzip'
+  depends_on 'fizmo'
 
   def install
     def copy_without(source_path, target_path, exclude)
@@ -27,7 +27,7 @@ class Zork1 < Formula
     copy_without('.', './share/zork1', 'share')
     file = File.new("zork1", "w+", 0755)
     file.write "#!/bin/bash
-jzip #{prefix}/share/zork1/DATA/ZORK1.DAT
+fizmo-console #{prefix}/share/zork1/DATA/ZORK1.DAT
 "
     bin.install "zork1"
     share.install "share/zork1"

--- a/zork2.rb
+++ b/zork2.rb
@@ -8,7 +8,7 @@ class Zork2 < Formula
   version '1'
   sha256 '7c360a14ec61d8e8f265e1aa99a13487c5e0f016a752e4b32314405a0e9ffea0'
 
-  depends_on 'jzip'
+  depends_on 'fizmo'
 
   def install
     def copy_without(source_path, target_path, exclude)
@@ -27,7 +27,7 @@ class Zork2 < Formula
     copy_without('.', './share/zork2', 'share')
     file = File.new("zork2", "w+", 0755)
     file.write "#!/bin/bash
-jzip #{prefix}/share/zork2/DATA/ZORK2.DAT
+fizmo-console #{prefix}/share/zork2/DATA/ZORK2.DAT
 "
     bin.install "zork2"
     share.install "share/zork2"

--- a/zork3.rb
+++ b/zork3.rb
@@ -8,7 +8,7 @@ class Zork3 < Formula
   version '1'
   sha256 'ef4a3b2bb7d3ae65de54a54520b550f13b3a2e964985550ffd401c8925bd0b10'
 
-  depends_on 'jzip'
+  depends_on 'fizmo'
 
   def install
     def copy_without(source_path, target_path, exclude)
@@ -27,7 +27,7 @@ class Zork3 < Formula
     copy_without('.', './share/zork3', 'share')
     file = File.new("zork3", "w+", 0755)
     file.write "#!/bin/bash
-jzip #{prefix}/share/zork3/DATA/ZORK3.DAT
+fizmo-console #{prefix}/share/zork3/DATA/ZORK3.DAT
 "
     bin.install "zork3"
     share.install "share/zork3"

